man-roxygen/endpoint.R

```diff
@@ -0,0 +1 @@
+#' @param endpoint Which Travis endpoint (.org / .com) to use. Default is '.org'.
```

Comment:
I wonder if this is a setting the user should be able to set on a repo-by-repo basis or globally. Can we detect it?

Reply:
We cannot check as one can even have both enabled at the same time. This is possible if one has not yet pressed the "migrate" button on Travis.com.

For global setting, see the solution proposed in #115

Reply:
I think we should only support it on a global level. Why? Users who already migrated fully won't be able to use .org at all anymore. At some point travis will force the transition and then .org becomes fully deprecated. If we have per-repo settings by then, users have to clean up way more than making a single global change.

Typically, users have their repos (at least mainly) on either .org or .com so this approach should work for most.

Reply:
Globally would be great. An option that sets the default doesn't seem to be too bad here.